types/d3-chord/index.d.ts

```diff
@@ -473,3 +473,30 @@ export function ribbon<Datum, SubgroupDatum>(): RibbonGenerator<any, Datum, Subg
  * The third generic corresponds to the datum type of the chord subgroup, i.e. source or target of the cord. The default type is ChordSubgroup.
  */
 export function ribbon<This, Datum, SubgroupDatum>(): RibbonGenerator<This, Datum, SubgroupDatum>;
+
+/**
+ * Creates a new arrow ribbon generator with the default settings.
+ */
+export function ribbonArrow(): RibbonGenerator<any, Ribbon, RibbonSubgroup>;
```

Comment:
Can  you make a `RibbonArrowGenerator` that has [the `headRadius` property](https://github.com/d3/d3-chord#ribbonArrow_headRadius) which `ribbonArrow()` would then return?

Reply:
Ah right, I thought so

Reply:
Ok I tried to add it. Not sure what to write in docs though. Should I copy paste it from https://github.com/d3/d3-chord?